content/docs/guides/continuous-delivery/spinnaker.md

```diff
@@ -101,98 +126,126 @@ To create this resource in your Spinnaker cluster, run:
+instead of `preview`. In the end, your pipeline should look like this:
 
 ![Full Pipeline](/images/docs/guides/continuous-delivery/spinnaker/full-pipeline.png)
 
 Go back to the main Pipelines view and click on the **Start Manual Execution** link next to your pipeline.
-Check the logs from Pulumi as the Pulumi stage runs. The logs update in near real-time.
+Once the job starts execution, you should see a **Console Output** link. Clicking it will open a modal that will show
+you the current output.
 
-![Pulumi Stage](/images/docs/guides/continuous-delivery/spinnaker/pipeline-logs.png)
+**Note**: If you are running Spinnaker on your machine, it can take a while before the job starts to execute and the logs
+are visible. If you see an error about `ContainerCreating` it means the pod running the job isn't ready yet.
 
-Once the pipeline is complete, assuming you have not modified the example app, the `nginx` container would have been deployed to the `default` namespace.
-Get the pod IP by running a `kubectl` command and `curl`-ing the IP to see the response:
+Once the pipeline is complete, assuming you have not modified the example app, the `nginx` container would have been
+deployed to the `default` namespace. Get the pod IP by running a `kubectl` command and `curl`-ing the IP to see the response:
 
 ```
 kubectl get pods -o wide
 
-curl http://POD_IP
+curl http://<POD_IP>
 ```
 
+## FAQs
+
+**Can I get colored log output from Pulumi?**
+
+Yes! Simply pass the Pulumi CLI arg `--color always`. Now when you open the `Console Output` modal, you should see
+colors in the log output. Note that the colored logs are only visible starting with v0.2.0 of the plugin.
+
+**How do I use the plugin with an enterprise VCS repo such as GitHub Enterprise?**
+
+The plugin allows you to specify custom container images. One option is to build the container image with the SSH
+credentials baked-in. This allows you to specify a "git SSH" URL. Consult with your Spinnaker administrator before you
+do this.
+
+The other option is to create a Personal Access Token and create a Secret resource that you reference in your stage.
+You can then use the key names as placeholders in the HTTPS version of your repo URL.
+
+**How do I specify my cloud credentials?**
+
+As mentioned before you can define a `Secrets` resource whose key/value pairs will be mapped as environment variables.
+The key names need to match the environment variables that a cloud provider supports.
+
+The other option is similar to baking the container image with SSH credentials. You can also bake any configuration
+files used by the cloud provider in the container image. For example, `~/.aws/config` and `~/.aws/credentials`.
+You also have the option to override stack configuration from the pipeline stage by passing the `-c` flag and specify
+certain cloud provider settings that way. Again, taking AWS as an example, `-c aws:profile <profile name from ~/.aws/config>`.
```

Comment:
I think similar to your above question on baking things into the container image, I would take a more generalized approach to explaining this. I think generally speaking, the common path does not bake any sort of credentials into the image. Of course, we know folks who want to do this. Instead, I would include a sentence or two in the section about using a custom image to mention that in some cases, you may want to do so if you need certain materials available within that image (whether it's credentials or something else your CD process relies upon).

Reply:
I like the suggestion! I'll make the updates.

Reply:
I have applied the suggestions to the FAQs. Could you give it another read please?